crates/jsonschema-referencing/src/resolver.rs

```diff
@@ -156,7 +162,7 @@ impl<'r> Resolver<'r> {
     /// # Errors
     ///
     /// Returns an error if the resource id cannot be resolved against the base URI of this resolver.
-    pub fn in_subresource(&self, subresource: ResourceRef) -> Result<Self, Error> {
+    pub fn in_subresource(&self, subresource: &ResourceRef) -> Result<Self, Error> {
```

Comment:
it might be nice to have some `AsRef` trait here instead as `ResourceRef` is not 100% required, i.e. JSON content + draft are enough